src/data/observability-packs.json

```diff
@@ -1,17 +1,42 @@
 [
   {
-    "authors": [
-      "New Relic",
-      "Jakub Kotkowiak"
-    ],
+    "authors": ["New Relic", "Jakub Kotkowiak"],
     "dashboards": [
       {
         "description": "",
         "name": "Apache",
         "screenshots": [
-          "https://raw.githubusercontent.com/newrelic/newrelic-observability-packs/v0.8.2/packs/apache/dashboards/apache01.png",
-          "https://raw.githubusercontent.com/newrelic/newrelic-observability-packs/v0.8.2/packs/apache/dashboards/apache02.png",
-          "https://raw.githubusercontent.com/newrelic/newrelic-observability-packs/v0.8.2/packs/apache/dashboards/apache03.png"
+          "https://raw.githubusercontent.com/newrelic/newrelic-observability-packs/v0.8.2/packs/apache/dashboards/apache01.png"
+        ],
+        "url": "https://raw.githubusercontent.com/newrelic/newrelic-observability-packs/v0.8.2/packs/apache/dashboards/apache.json"
+      }
+    ],
+    "synthetics": [
```

Comment:
We don't want to commit fake data to this file since it currently mirrors the API.

Reply:
can do :+1:

Reply:
undid changes